cloudflared/rootfs/etc/s6-overlay/s6-rc.d/cloudflared/run

```diff
@@ -7,9 +7,13 @@ declare default_config=/tmp/config.json
 
 if bashio::config.has_value 'tunnel_token' ; then
```

Comment:
@elcajon since we now check two config values, there might be a better way to "construct" the cloudflared excec string (since they also always share a lot). Any suggestions?